kolibri/core/content/hooks.py

```diff
@@ -49,11 +50,14 @@ def render_to_page_load_async_html(self):
             self.frontend_message_tag()
             + self.plugin_data_tag()
             + [
-                '<script>{kolibri_name}.registerContentRenderer("{bundle}", ["{urls}"], {presets});</script>'.format(
-                    kolibri_name="kolibriCoreAppGlobal",
+                '<template data-viewer="{bundle}">{data}</template>'.format(
```

Comment:
I have not actually found a definitive answer as to whether a CSP that disallows `unsafe-inline` affects `<script type="application/json">` or not - my assumption was that it would, but the only verification I got of that was Claude agreeing with me (so I take that with a pinch of salt).

Reply:
Since it isn't evaluated with a different `type` other than JS, it seems like it shouldn't be affected by `unsafe-inline`. But again this seems just fine so probably not worth changing